src/components/Illustration/Illustration.js

```diff
@@ -118,20 +118,18 @@ const IllustrationComponent = ({ source, pageLanguage, children, data }) => {
       : {};
 
   return (
-    <>
-      <div>
-        <img
-          className="cutcorners"
-          style={{
-            objectFit: 'cover',
-            cursor: 'pointer',
-            width: '100%',
-            height: '100%',
-          }}
-          src={thumbnailSrc}
-          onClick={() => setCanvasModal(true)}
-        ></img>
-      </div>
+    <div style={{ display: 'flex', flexDirection: 'column' }}>
+      <img
+        className="cutcorners"
+        style={{
+          objectFit: 'cover',
+          cursor: 'pointer',
+          width: '100%',
+          height: '100%',
+        }}
+        src={thumbnailSrc}
+        onClick={() => setCanvasModal(true)}
+      ></img>
```

Comment:
```suggestion
      />
```
Image is self closing